docs/tutorials/geomesa-blobstore-exif.rst

```diff
@@ -0,0 +1,170 @@
+This allows support for new file formats to be quickly integrated. GeoMesa comes with a handler for EXIF headers,
+which are included in a variety of image formats from digital cameras that will be demonstrated by this tutorial.
+Also demonstrated in this tutorial is querying the BlobStore's DataStore index for Blobs and demonstrating the two
+step query - fetch workflow users use to interact with the BlobStore.
+
+GeoMesa BlobStore EXIF Handler Installation Instructions
+--------------------------------------------------------
+
+As the FileHandlers use the Java Service Registry, they can be loaded at runtime by simply placing the jar
+for the handler into the existing classpath. For this tutorial, we assume the user is deploying
+the EXIF FileHandler to an existing GeoServer with the BlobStore web servlet. See instructions above for
+setting up the BlobStore in the prerequisites.
+
+To deploy the EXIF FileHandler run:
+
+.. code-block:: bash
+
+    $ tar -xzvf \
+      geomesa-$VERSION/dist/handlers/geomesa-blobstore-exif-handler-$VERSION.jar \
+      -C /path/to/tomcat/webapps/geoserver/WEB-INF/lib/
+
+If you are using GeoServer's built in Jetty web server:
+
+.. code-block:: bash
+
+    $ tar -xzvf \
+      geomesa-$VERSION/dist/handlers/geomesa-blobstore-exif-handler-$VERSION.jar\
+      -C /path/to/geoserver/webapps/geoserver/WEB-INF/lib/
+
+Now the BlobStore is able to read image files with EXIF metadata headers.
+
+
+Using the EXIF Handler
+----------------------
+In order to use our new handler, we will need to get some geotagged images.
+Below is a partial Python script that uses the Flickr API to grab some geotagged images taken around Monument Valley.
+Using the Flickr API is beyond the scope of this tutorial, however the Python code snippet is provided for those
+interested in obtaining a test data set. The below script will download 100 images to the user's home directory.
+
+.. code-block:: python
+
+    import flickrapi
+    import requests
+    import itertools
+    api_key = u'Some API Key'
+    secret  = u'some secret'
+
+    flickr = flickrapi.FlickrAPI(api_key, secret)
+    flickr.authenticate_console(perms='read')
+    photos = flickr.photos_search(lat='37.0000', lon='-110.1700', radius='10', safe_search='1', extras='url_o')
+
+    urls = itertools.islice((p.attrib['url_o'] for p in photos[0] if 'url_o' in p.keys()), 100)
+
+    for i, url in enumerate(urls):
+        with open('~/flickr{0:03d}.jpg'.format(i), 'wb') as fd:
+            r = requests.get(url)
+            for chunk in r.iter_content(1024):
+                fd.write(chunk)
+
+
+
+First we need to register a BlobStore using the following command:
+
+.. code-block:: bash
+
+    $ curl -d 'instanceId=myCloud' -d 'zookeepers=zoo1,zoo2,zoo3' -d 'tableName=myblobstore' -d 'user=user' -d 'password=password' http://localhost:8080/geoserver/geomesa/blobstore/ds/myblobstore
```

Comment:
indicate above that user/pwd/etc need to be correct for your environment, otherwise people will blindly copy/paste